homework/index.js

```diff
@@ -29,10 +29,68 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function changeDateFormat(date) {
+    const timeFormat = new Date(date);
+    return timeFormat.toLocaleString();
+  }
+
+  function sortRepos(a, b) {
+    const first = a.name.toLowerCase();
+    const second = b.name.toLowerCase();
+    const key = first < second ? -1 : first > second ? 1 : 0;
+    return key;
   }
 
+  function renderRepoDetails(repo, ul) {
+    const grouping = createAndAppend('li', ul, { class: 'listItem' });
+    const name = createAndAppend('li', grouping, {
+      text: 'Repository name: ',
+      class: 'title',
+    });
+    const repoLang = createAndAppend('li', grouping, {
+      text: 'Repository language: ',
+      class: 'title',
+    });
+    const description = createAndAppend('li', grouping, {
+      text: 'Description: ',
+      class: 'title',
+    });
+    const forkCount = createAndAppend('li', grouping, {
+      text: 'Fork count: ',
+      class: 'title',
+    });
+    const createTime = createAndAppend('li', grouping, {
+      text: 'Created on: ',
+      class: 'title',
+    });
+    const updateTime = createAndAppend('li', grouping, {
+      text: 'Updated on: ',
+      class: 'title',
+    });
+
+    createAndAppend('a', name, {
+      text: repo.name,
+      href: repo.html_url,
+      class: 'content',
+    });
+    createAndAppend('span', repoLang, {
+      text: repo.language,
+      class: 'content',
+    });
+    createAndAppend('span', description, {
+      text: repo.description,
+      class: 'content',
+    });
+    createAndAppend('span', forkCount, { text: repo.forks, class: 'content' });
+    createAndAppend('span', createTime, {
+      text: changeDateFormat(repo.created_at),
+      class: 'content',
+    });
+    createAndAppend('span', updateTime, {
+      text: changeDateFormat(repo.updated_at),
+      class: 'content',
+    });
+  }
   function main(url) {
```

Comment:
For readability, place a blank line before and after each function definition. It's like separating paragraphs with a blank line in a piece of text: it improves readability.

Reply:
Right..